spec/index.bs

```diff
@@ -2885,6 +2885,7 @@ enum GPUFeatureName {
     "clip-distances",
     "dual-source-blending",
     "subgroups",
+    "interited-bundle-bind-groups",
```

Comment:
```suggestion
    "inherited-bundle-bind-groups",
```

Though it likely can be a core feature instead!

Reply:
Fixed the typo everywhere, thanks for catching that! And it would be great if this could be part of core! I'll continue treating it as feature, though, till we get group buy-in on it.